lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1280,6 +1280,112 @@ void BoundInterpreterFunction::fwdBatchOneHotInst(
   }
 }
 
+// Example:
```

Comment:
Thank you for great illustration.
In general, we don't have examples in this file. Could you please move this comment to a unit test, and make a unit test out of it? Maybe use blocksize=3.